CSharpFunctionalExtensions/CSharpFunctionalExtensions.csproj

```diff
@@ -47,4 +47,9 @@
     </PackageReference>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)'=='net40' OR '$(TargetFramework)'=='net45' OR '$(TargetFramework)'=='net461' OR '$(TargetFramework)'=='net472' OR '$(TargetFramework)'=='netstandard2.0'">
+    <Compile Remove="Result\Dto\**" />
+    <EmbeddedResource Remove="Result\Dto\**" />
+    <None Remove="Result\Dto\**" />
+  </ItemGroup>
```

Comment:
As I see `Result\Dto\**` is not the base name for folders now.

Let's create a new folder `Serialization\Json` and place all files there

Reply:
Json\Serializaion was created (same as sysmte,text.json.serialziation)